src/field_region_map.c

```diff
@@ -45,6 +49,12 @@ static void VBCB_FieldUpdateRegionMap(void);
 static void MCB2_FieldUpdateRegionMap(void);
 static void FieldUpdateRegionMap(void);
 static void PrintRegionMapSecName(void);
+static void PrintTitleWindowText(void);
+
+static const u8* const FlyPromptText = COMPOUND_STRING("{R_BUTTON}FLY");
```

Comment:
This should be just a normal text string. We don't use compound strings like that.

Reply:
this is actually straight from the strings used in the pokenav helpbar, though there it's an array.

Reply:
We use Compound Strings for non duplicate strings, so single use strings. If you want this to use multiply times it should be a normal string so
```
static const u8 sFlyPropmtText[] = _("{R_BUTTON}FLY");
```

Reply:
this is technically a single use string- should it stay a compound string then?

Reply:
You use it twice right?

The reason we use compound strings is because they make code more readable and additionally save a little bit of space so in your case it should be moved where it is needed in the first place. Though iirc we've decided to keep using normal strings for function arguments so you would need to convert it either way.

Reply:
oh, i see, yeah it's used once in the file, but twice in the function. fixed!

Reply:
hmm in this case should it be defined in the `PrintTitleWindowText` func? since it's only used there?

Reply:
yes

Reply:
done!